test/core/bin.js

```diff
@@ -132,7 +132,12 @@ _.each(fs.readdirSync(fixtureLoc), function (binName) {
       };
 
       var optionsLoc = testLoc + "/options.json"
-      if (fs.existsSync(optionsLoc)) _.merge(opts, require(optionsLoc));
```

Comment:
You can run Babel in the browser. `make test-browser` will fail (in fact I just ran it and it fails on two tests oddly) because it serialises all the tests to JSON and so the `expected.js` wont match since the function wont be serialised.